streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1188,6 +1187,7 @@ public void start(Application appParam, boolean auto) throws Exception {
 
         assert application != null;
 
+        application.setJobId(new JobID().toHexString());
```

Comment:
If the jobid is generated actively, then the copy method in ApplicationServiceImpl class also needs to add the code of creating new jobid, for example: 
```
newApp.setJobId (new JobID (). ToHexString ())
```

Reply:
setJobId is only executed at start job and does not affect the copy logic

Reply:
> If the jobid is generated actively, then the copy method in ApplicationServiceImpl class also needs to add the code of creating new jobid, for example:
> 
> ```
> newApp.setJobId (new JobID (). ToHexString ())
> ```

@youyangkou  setJobId is only executed at start job and does not affect the copy logic